src/ServiceProviders/LegacyServiceProvider.php

```diff
@@ -264,6 +264,8 @@ private function is_request($type)
                 return ( ! is_admin() || defined('DOING_AJAX')) && ! defined('DOING_CRON') && ! defined('REST_REQUEST');
             case 'wpcli':
                 return defined('WP_CLI') && WP_CLI;
+            default:
+                return false;
```

Comment:
Unless this is intended behavior, throw an exception.

Reply:
On second look this is intended to fail silently to prevent an error, but such an error would be a result of developer error.

I pushed an update to enforce the supported request types, see [86fe7b8](https://github.com/impress-org/givewp/pull/6513/commits/86fe7b8102c0a305618bc6e5f7d79073c20cf38c).